README.md

```diff
@@ -385,16 +385,27 @@ K9s uses aliases to navigate most K8s resources.
     refreshRate: 2
     # Number of retries once the connection to the api-server is lost. Default 15.
     maxConnRetry: 5
-    # Enable mouse support. Default false
-    enableMouse: true
-    # Set to true to hide K9s header. Default false
-    headless: false
-    # Set to true to hide K9s crumbs. Default false
-    crumbsless: false
     # Indicates whether modification commands like delete/kill/edit are disabled. Default is false
     readOnly: false
     # Toggles whether k9s should exit when CTRL-C is pressed. When set to true, you will need to exist k9s via the :quit command. Default is false.
     noExitOnCtrlC: false
+    #UI settings
+    ui:
+      # Enable mouse support. Default false
+      enableMouse: false
+      # Set to true to hide K9s header. Default false
+      headless: false
+      # Set to true to hide the K9S logo Default false
+      logoless: false
+      # Set to true to hide K9s crumbs. Default false
+      crumbsless: false
+      noIcons: false
+      # Toggles reactive UI. This option provide for watching on disk artifacts changes and update the UI live Defaults to false.
+      reactive: false
+      # By default all contexts wil use the dracula skin unless explicitly overridden in the context config file.
+      skin: dracula # => assumes the file skins/dracula.yaml is present in the  $XDG_DATA_HOME/k9s/skins directory
+      # By default start all Live_View in full screen mode. (yaml, helm history, describe, value_extender) Default false
+      fullScreenLView: false
```

Comment:
Wondering if we could use a  a better name for this?? 
Perhaps `fullScreen` or `defaultsToFullScreen`  or `activateFullScreen` might be better choices?? 
Likely if users opt to default to FS likely the logs should operate as such. So I am thinking we should axe the log fullScreen and go with a single option to configure this ui bit in just one place??
What do you think?

Reply:
I think having a defaultsToFullScreen option under Ui which sets all live/details view and logs is a better option.

For the naming I don't like `fullScreenLView` but `fullScreen ` or  `defaultsToFullScreen ` in UI sounds like it is about the whole K9S not just the "popup" views. Since I could not come up with a better one I will just use defaultsToFullScreen.

Reply:
Thanks for the feedback Adam! Good point! I think that make sense. 
Likely in the future we may end up breaking out sub-sections under UI for details view configs.